homeassistant/components/litejet/__init__.py

```diff
@@ -37,27 +36,13 @@
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up the LiteJet component."""
-    if DOMAIN in config and not hass.config_entries.async_entries(DOMAIN):
+    if DOMAIN in config:
         # No config entry exists and configuration.yaml config exists, trigger the import flow.
```

Comment:
Nit: this comment isn't accurate and should now be more like `# A configuration.yaml config exists, trigger the import flow.`